tests/negative/proxy/http.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package proxy
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/coreos/ignition/tests/register"
+	"github.com/coreos/ignition/tests/types"
+)
+
+var (
```

Comment:
Move this to `tests/servers.go`

Reply:
@ajeddeloh When you get a chance, could you clarify? Are you asking me to move everything under

```golang
var (
...
)
```

to `tests/servers.go`?

Reply:
I was trying to follow the same convention as timeouts and tls tests.

Reply:
I was, yeah. I didn't realize those tests put them with the test. I guess this ok in that case.